pydatastructs/graphs/algorithms.py

```diff
@@ -188,7 +190,7 @@ def _breadth_first_search_parallel_adjacency_list(
                 status = Executor.submit(
                          _generate_layer, args=
                          (graph, node, layers[layer+1], visited,
-                          operation, *args), kwargs=kwargs).result()
+                          operation, args), kwargs=kwargs).result()
```

Comment:
```suggestion
                          operation, *args), kwargs=kwargs).result()
```
Please avoid irrelevant changes.

Reply:
Yes, I was having pytest of older version installed, hence it gave error in these lines. Now the problem is rectified, I will modify it back.